packages/cli/demo/README.md

```diff
@@ -258,11 +258,16 @@ _Familiar Chat_ is an example application that provides a web app for
 interacting with your pet daemon.
 
 ```
-> endo open familiar-chat cat.js --powers HOST
+> endo install familiar-chat cat.js --powers HOST
```

Comment:
I could see an argument that this might be a more predictable usage if shaped more like other commands: `endo install cat.js --powers HOST --name familiar-chat`, but the `--name` flag is not optional for `install`. If that’s preferable to others, I’d consider doing that in a follow-up change.